eng/docs/index/templates/matthews/partials/class.tmpl.partial

```diff
@@ -26,7 +26,7 @@
 <div id="collapsible-{{id}}" class="collapse in">
 <div class="markdown level1 summary">{{{summary}}}</div>
 <div class="markdown level1 conceptual">{{{conceptual}}}</div>
-<h5 class="decalaration">{{__global.declaration}}</h5>
```

Comment:
@danieljurek should I revert this change and wait for [this](https://github.com/MathewSachin/docfx-tmpl/pull/4) to get merged instead or is it OK to fix this here?

Reply:
No problem with merging this right now as we're not doing doc index generation today. I have added this to the list of things to do in this issue: https://github.com/Azure/azure-sdk-for-c/issues/930